patches/minecraft/net/minecraft/world/effect/MobEffect.java.patch

```diff
@@ -17,12 +17,22 @@
     }
  
     public void m_6742_(LivingEntity p_19467_, int p_19468_) {
-@@ -179,4 +_,29 @@
+@@ -179,4 +_,39 @@
     public boolean m_19486_() {
        return this.f_19447_ == MobEffectCategory.BENEFICIAL;
     }
 +
 +   // FORGE START
++   private final net.minecraftforge.common.util.ReverseTagWrapper<MobEffect> reverseTags = new net.minecraftforge.common.util.ReverseTagWrapper<>(this, () -> net.minecraft.tags.SerializationTags.m_13199_().m_144452_(Registry.f_122900_));
++
++   public java.util.Set<net.minecraft.resources.ResourceLocation> getTags() {
++      return reverseTags.getTagNames();
++   }
++
++   public boolean is(net.minecraft.tags.Tag<MobEffect> tag) {
++      return tag.m_8110_(this);
```

Comment:
These patches can be smaller if you move the braces up all on one line

Reply:
No need to one-line things. what we want is to minimize the *complexity* of the patches, not the length per se.

Reply:
Ah, I had had a similar PR and someone told me this so that is good to know